smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -179,10 +179,10 @@ private void generateOperationSerializer(
         // Add the normalized input type.
         Symbol inputType = symbol.expectProperty("inputType", Symbol.class);
 
-        writer.openBlock("export function $L(\n"
+        writer.openBlock("export async function $L(\n"
```

Comment:
Similar changes needed in [`HttpRpcProtocolGenerator`](https://github.com/awslabs/smithy-typescript/blob/master/smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpRpcProtocolGenerator.java#L122).

Reply:
Added. @kstich